tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./public/*.html",
+    "./app/assets/**/*",
+    "./app/views/**/*.rb",
+    "./app/tempates/**/*",
```

Comment:
You have a typo here

```suggestion
    "./app/templates/**/*",
```

Reply:
Well spotted, thanks!